subsys/fs/fs.c

```diff
@@ -601,6 +603,10 @@ int fs_stat(const char *abs_path, struct fs_dirent *entry)
 	}
 
 	rc = mp->fs->stat(mp, abs_path, entry);
+	if (rc == -ENOENT) {
+		/* File doesn't exist, which is a valid stat response */
+		return rc;
+	}
```

Comment:
Can you also make it `else if` as in the case of `fs_mkdir` ?

Reply:
Like I have done it now?